content/en/docs/architecture/jira.md

```diff
@@ -22,6 +22,22 @@ To rerun the validations, comment `/jira refresh` on the PR. This is mostly usef
 All Openshift bugs must be made in the `OCPBUGS` project. For PRs referencing issues from other projects, a link will be
 created but the rest of the lifecycle management will not be performed for the issue.
 
+## Premerge Verification
+
+The Jira GitHub integration provides a method to mark a PR as `verified` before the PR is merged. This allows the integration
+to move the bug to the `VERIFIED` state upon merge instead of requiring a QE engineer to test and manually move an issue to the
+`VERIFIED` state after the issue moves into the `ON_QA` state. Premerge verfication is strongly preferred.
+
+To mark a PR as `verified`, comment `/verified by reason` on the PR. The reason can be the names of tests, GitHub usernames,
+documentation, or links. Multiple reasons can be listed, separated by a comma. Example: `/verified by @user` or
+`/verified by test1,test2`. If successful, the bot will comment on the PR and add the `verified` label to the PR.
+
+In certain situations, issues may need to be verified after the PR is merged. In these cases, the `/verified later` command
+can be used. The `/verified later` command takes a username as a field (example: `/verified later @username`). This will add
+the `verified-later` tag to the PR. Upon merge, the PR will be moved to the `MODIFIED` state and then `ON_QA` once a nightly
```

Comment:
Assuming QE finds a way, will a subsequent `/verified by` remove `verified-later` ?

Reply:
If a `/verified by` command is issued after a `/verified later`, both labels will exist on the PR. This doesn't really matter for the plugin itself as it only cares about the `verified` label when merging, but I can update the plugin to remove the `verified-later` label as well.